frontend/src/routes/(app)/(home)/reports/+page.svelte

```diff
@@ -4,7 +4,7 @@
 	export let data;
 </script>
 
-<div class="flex flex-wrap mb-6 overflow-y-scroll h-full content-start">
```

Comment:
Are these still scrolling? They should, right?

Reply:
Yup it scrolls the whole panel now, better on mobile and works on desktop

Reply:
Looking at in on hub I think this is a regression. The whole page now scrolls which makes the header go away. I think we should revert this.

Reply:
![Uploading image.png…]()